src/main/java/at/hannibal2/skyhanni/features/fishing/tracker/SeaCreatureTracker.kt

```diff
@@ -156,5 +164,12 @@ object SeaCreatureTracker {
     }
 
     private fun isEnabled() = LorenzUtils.inSkyBlock &&
-        FishingAPI.lastActiveFishingTime.passedSince() < 10.minutes && config.enabled
+        FishingAPI.lastActiveFishingTime.passedSince() < 10.minutes && config.enabled && !isTrophyFishing()
+
+    private fun isTrophyFishing(): Boolean {
+        val armorInternalNames = InventoryUtils.getArmor().map { it?.getInternalName()?.asString() ?: return false }
```

Comment:
The reutn false part can go: matches() returns false when the parameter is empty.

Reply:
Also you can remove `.map {` and move it in `.all {`